File: C/beginner/1007.c
/*Incluing Libraries*/
#include <stdio.h>

/*Main Function*/
int main() {
    /*Variable declaration and initialization*/
    int a, b, c, d, dif;
    
    /*Requesting values to user*/
    scanf("%d", &a);
    scanf("%d", &b);
    scanf("%d", &c);
    scanf("%d", &d);
    
    /*Doing the calculations*/
    dif = ((a * b) - (c * d));
    
    /*Showing results*/
    printf("DIFERENCA = %d\n", dif);

    return 0;
}

/*By Fernando Serra*/
/*https://github.com/fernandosserra*/

File: Haskell/beginner/1007.hs
-- Doing imports
import Text.Printf

-- Main Function
main = do
    -- Requesting values and assign into variables
    a <- readLn :: IO Int
    b <- readLn :: IO Int
    c <- readLn :: IO Int
    d <- readLn :: IO Int
    
    -- Doing the calculations
    let dif = (a * b) - (c * d)

    -- Showing results
    printf "DIFERENCA = %d\n" dif

-- By Fernando Serra
-- https://github.com/fernandosserra